doc/source/report/overview.rst

```diff
@@ -148,7 +148,38 @@ whole project can be divided into 6 sub-projects:
 -  **Strain:** A strain here is a genetic variant or sub-type of of the
    more general mouse population.
 
-5. References
+5. Data 
+-------
+The original data includes two folders, intervals and txy_coords, and a npy file named all_features_mousedays_11bins.
+The all_features_mousedays_11bins is a 9*1921*11 matrix, which represents 9 features among 1921 mouse days in 11 2 hour
```

Comment:
You should mark file names using fixed width fonts.  I would also use the actual filename including its extension.  I would also say `contains` instead of `is` since it isn't actually a matrix, but rather a file containing the information needed to create a matrix.